client/src/app/shared/components/media-upload-content/media-upload-content.component.html

```diff
@@ -107,10 +118,10 @@
         color="primary"
         [disabled]="uploadList.data.length === 0"
     >
-        <span translate> Upload </span>
+        <span>{{ ' Upload ' | translate }}</span>
```

Comment:
Please remove all blanks in translateable strings. Otherwise the extract tool identified it as new string in template.pot